translation-service-etranslation/src/main/java/eu/europeana/api/translation/service/etranslation/RedisMessageListener.java

```diff
@@ -21,17 +21,22 @@ public void onMessage(Message message, byte[] pattern) {
           LOGGER.debug("New message received from RedisMessageListener: {}", message);
         }
         String messageBody=new String(message.getBody(), StandardCharsets.UTF_8);
+        if(messageBody.contains(ETranslationTranslationService.eTranslationErrorCallbackIndicator)) {
+          //if we enter here, means the eTranslation error callback is called
+          this.message=null;
```

Comment:
we should throw and exception in this case, cause the API to return an error response